internal/pkg/peer/blocksprovider/bft_deliverer.go

```diff
@@ -46,17 +47,7 @@ type DurationExceededHandler interface {
 
 //go:generate counterfeiter -o fake/updatable_block_verifier.go --fake-name UpdatableBlockVerifier . UpdatableBlockVerifier
 type UpdatableBlockVerifier interface {
```

Comment:
why not `type UpdatableBlockVerifier deliverclient.CloneableUpdatableBlockVerifier` ?

Reply:
that works too